src/routes/api/[...endpoint]/proxy.ts

```diff
@@ -12,7 +13,7 @@ async function _fetchApi(
   headers ??= new Headers();
   headers.append('Content-Type', 'application/json');
   headers.append('Accept', 'application/json');
-  if (window?.location?.origin) {
+  if (browser && window?.location?.origin) {
```

Comment:
when is the condition needed?

Reply:
I was getting an error without this when submitting the forms:
`ReferenceError: window is not defined
    at _fetchApi (/home/rita/Documents/ni_site/website-niaefeup-frontend/src/routes/api/[...endpoint]/proxy.ts:16:3)
    at Module.fetchWithAuth (/home/rita/Documents/ni_site/website-niaefeup-frontend/src/routes/api/[...endpoint]/proxy.ts:44:12)
    at dispatchToBackend (/home/rita/Documents/ni_site/website-niaefeup-frontend/src/routes/api/[...endpoint]/+server.ts:12:10)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Module.render_endpoint (/home/rita/Documents/ni_site/website-niaefeup-frontend/node_modules/@sveltejs/kit/src/runtime/server/endpoint.js:43:18)
    at async resolve (/home/rita/Documents/ni_site/website-niaefeup-frontend/node_modules/@sveltejs/kit/src/runtime/server/respond.js:446:17)
    at async Object.handle (/home/rita/Documents/ni_site/website-niaefeup-frontend/src/hooks.server.ts:27:20)
    at async Module.respond (/home/rita/Documents/ni_site/website-niaefeup-frontend/node_modules/@sveltejs/kit/src/runtime/server/respond.js:322:20)
    at async file:///home/rita/Documents/ni_site/website-niaefeup-frontend/node_modules/@sveltejs/kit/src/exports/vite/dev/index.js:524:22`

Reply:
An article about a similar error: https://dev.to/vvo/how-to-solve-window-is-not-defined-errors-in-react-and-next-js-5f97
I think this is a fine fix